### ISSUE TITLE
magic numbers are hard to follow and error prone

### ISSUE THREAD

**Issue by incredibleLeitman** [nsofiia/slot_machine#9]
This line is only understandable if the reader knows, that you are having a 3x3 grid and want to have a break after each row
https://github.com/nsofiia/slot_machine/blob/cc84a2905430abaafe79aac3fa996d7f5db758fc/Program.cs#L92

You can instead use the amount of cols, which would also allow your program to run with other grid sizes than 3x3 without changing any code :D

**Comment by nsofiia** [comment]
@incredibleLeitman fixed, ready for review
